swatch-metrics/deploy/clowdapp.yaml

```diff
@@ -0,0 +1,505 @@
+                    key: password
+              - name: CLOUDIGRADE_ENABLED
+                value: ${CLOUDIGRADE_ENABLED}
+              - name: CLOUDIGRADE_PSK
+                valueFrom:
+                  secretKeyRef:
+                    name: cloudigrade-psk
+                    key: psk
+              - name: PROM_URL
+                value: ${PROM_URL}
+              - name: OPENSHIFT_BILLING_MODEL_FILTER
+                value: ${OPENSHIFT_BILLING_MODEL_FILTER}
+              - name: USER_HOST
+                value: ${USER_HOST}
+              - name: USER_MAX_CONNECTIONS
+                value: ${USER_MAX_CONNECTIONS}
+              - name: USER_MAX_ATTEMPTS
+                value: ${USER_MAX_ATTEMPTS}
+              - name: USER_BACK_OFF_MAX_INTERVAL
+                value: ${USER_BACK_OFF_MAX_INTERVAL}
+              - name: USER_BACK_OFF_INITIAL_INTERVAL
+                value: ${USER_BACK_OFF_INITIAL_INTERVAL}
+              - name: USER_BACK_OFF_MULTIPLIER
+                value: ${USER_BACK_OFF_MULTIPLIER}
+              - name: RHSM_KEYSTORE_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: tls
+                    key: keystore_password
+              - name: RHSM_KEYSTORE
+                value: /pinhead/keystore.jks
+              - name: DEV_MODE
+                value: ${DEV_MODE}
+              - name: SWATCH_SELF_PSK
+                valueFrom:
+                  secretKeyRef:
+                    name: swatch-psks
+                    key: self
+            livenessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /health/liveness
+                port: 9000
+                scheme: HTTP
+              initialDelaySeconds: 60
+              periodSeconds: 20
+              successThreshold: 1
+              timeoutSeconds: 5
+            readinessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /health
+                port: 9000
+                scheme: HTTP
+              initialDelaySeconds: 60
+              periodSeconds: 20
+              successThreshold: 1
+              timeoutSeconds: 5
+            resources:
+              requests:
+                cpu: ${CPU_REQUEST}
+                memory: ${MEMORY_REQUEST}
+              limits:
+                cpu: ${CPU_LIMIT}
+                memory: ${MEMORY_LIMIT}
+            volumeMounts:
+              - name: logs
+                mountPath: /logs
+              - name: pinhead
+                mountPath: /pinhead
+            volumes:
+              - name: logs
+                emptyDir:
+              - name: pinhead
+                secret:
+                  secretName: pinhead
+            sidecars:
+              - name: token-refresher
+                enabled: true
+
+      jobs:
+        - name: metrics-sync
+          schedule: ${METERING_SCHEDULE}
+          activeDeadlineSeconds: 4200
```

Comment:
This still needs to be lowered to 1800.

Reply:
Sorry, thought I left another item about this. Reason being, otherwise we get jobs running too long w/ sidecar and overlapping.